.packit.yaml

```diff
@@ -22,7 +22,7 @@ actions: &base-actions
     - hatch version
 
 targets: &all-targets
```

Comment:
Hmmm... just realized that `all-targets` is used for `propose_downstream` and `koji_build` jobs as well. This would mean that we would not get rawhide pull requests created for the dist git pagure.

Reply:
Agreed on the hacking session to disable both Rawhide testing and building for now.